exercise_7_answer_lu_zhang.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+import json
+
+import pandas as pd
+
+def check_path(path):
+
+
+
+def read_csv(file):
+    tbl = pd.read_csv(file)
+    print(len(tbl.index)+1)
+
+
+def write_csv(data_list, output_file):
+    obj = pd.DataFrame(data_list)
```

Comment:
this `pd.DataFrame(data_list)` may or may not work for the specified data structure, explore `csv.writer` as we discussed in class